src/assets/assets.json

```diff
@@ -0,0 +1,82 @@
+{"image": 
+ [{
+
+    "key":"sky",
+    "location":"http://labs.phaser.io/assets/skies/space3.png"
```

Comment:
Do you think calling these keys `path` instead of `location` will make it more readable?

Reply:
Yep